policy/modules/system/systemd.te

```diff
@@ -1527,8 +1541,19 @@ seutil_libselinux_linked(systemd_user_session_type)
 allow systemd_user_runtime_dir_t self:capability { fowner chown sys_admin dac_read_search dac_override };
 allow systemd_user_runtime_dir_t self:process setfscreate;
 
+manage_sock_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_notify_t, systemd_user_runtime_notify_t)
+
+list_dirs_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_dir_t, systemd_user_runtime_dir_t)
+read_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_dir_t, systemd_user_runtime_dir_t)
+
 domain_obj_id_change_exemption(systemd_user_runtime_dir_t)
 
+manage_dirs_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_t, systemd_user_runtime_t)
+manage_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_t, systemd_user_runtime_t)
+manage_sock_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_t, systemd_user_runtime_t)
+manage_fifo_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_t, systemd_user_runtime_t)
+manage_chr_files_pattern(systemd_user_runtime_dir_t, systemd_user_runtime_t, systemd_user_runtime_t)
+
```

Comment:
This shouldnt be needed with modern systemd (i guess this is for /run/user/UID/systemd/inaccessible)) in latest systemd this is managed by systemd it self and not systemd_user_runtime_dir_t) no sure if you want to support this "fluke" as it only existed for one systemd release and was quickyl fixed thereafter

Reply:
Accesses for these were resolved with the proper access for `systemd_user_runtime_dir_t` to delete user runtime files.